metadata_backend/api/operators.py

```diff
@@ -567,6 +589,20 @@ async def _format_read_data(self, schema_type: str, data_raw: Dict) -> str:
         """
         return data_raw["content"]
 
+    async def check_exists(self, schema_type: str, accession_id: str) -> None:
```

Comment:
`Operator(BaseOperator)` and `XMLOperator(BaseOperator)` both use `check_exists()` and they both inherit `BaseOperator`, so can `check_exists()` be moved to `BaseOperator` class, so that the function is not duplicated?

Reply:
it is an a `ABC` https://docs.python.org/3/library/abc.html and the method needs to be implemented in the subclass